creating_geometric_models/triangulate_point_cloud_and_clean_up.py

```diff
@@ -0,0 +1,101 @@
+from sklearn.decomposition import PCA
```

Comment:
The file should begin with a shebang like
#!/usr/bin/env python3
and should have the executable flag set (chmod +x triangulate...)